pkg/config/env/env_variables.go

```diff
@@ -23,4 +23,6 @@ const (
 	OtlpExporterInsecure string = "OTEL_EXPORTER_OTLP_INSECURE"
 	// OpenTelemetry transport protocol (grpc, http/protobuf, http/json)
 	OtlpExporterProtocol string = "OTEL_EXPORTER_OTLP_PROTOCOL"
+	// OpenTelemetry headers to add to transport
+	OTEL_EXPORTER_OTLP_HEADERS string = "OTEL_EXPORTER_OTLP_HEADERS"
```

Comment:
env var defined here

https://opentelemetry.io/docs/specs/otel/protocol/exporter/#specifying-headers-via-environment-variables

Reply:
The lint warning relates to the variable name `OTEL_EXPORTER_OTLP_HEADERS string`, should be `OtlpExporterHeaders string`